kotlin-website.py

```diff
@@ -48,6 +48,7 @@
 build_check_links = True
 build_errors = []
 url_adapter = app.create_url_adapter(None)
+node_env = os.environ.get('NODE_ENV', 'development')
```

Comment:
Do we set "production" for the variable in prod or we need to do that?

Reply:
No, it is a new variable only considered in development layer. So at the moment it doesn't matter do we set it's value in production or not. The only possible issue is a little inconsistency — there is value in dev and no value in prod. But nothing will break